tests/test_views.py

```diff
@@ -51,7 +50,9 @@ def test_query_matching_authors(self):
         # Create ten citations to match the query and one that doesn't.
         factories.CitationFactory.create_batch(10, author='Verne, Jules')
         factories.CitationFactory(author='Dan Longshot')
```

Comment:
I don't think it's too much. I think it's a reasonable thing to test for. And it will be simple to add.